content/kots-cli/remove.md

```diff
@@ -0,0 +1,32 @@
+---
+date: 2020-11-20
+linktitle: "remove"
+title: kots remove
+weight: 90120
+---
+
+Remove application reference from Admin Console.
+Multiple applications can be installed using the `Add a new application` button in Admin Console or using the `kots install` command.
+All installed applications will appear in Admin Console.
+The `kots remove` command can be used to remove them.
+Note that deployed application will not be removed from the cluster, only it's reference will be removed from Admin Console.
+
+### Usage
+```bash
+kubectl kots remove [app-slug]
+```
+* _Replace `[app-slug]` with the slug for the KOTS application that should be removed (required)._
+* _If the KOTS application has been packaged by Replicated Vendor, the `--license-file` flag must be provided._
```

Comment:
Why is the license file required on remove? Is that just for a safety check? Seems burdensome. Also, how would a Kots application not be packaged by Replicated Vendor? Why is Vendor capitalized? Needs a "the".